content/en/sidecar-container.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Sidecar Container
+status: Completed
```

Comment:
```suggestion
status: Feedback Appreciated
```

When you create a new term, please set it to "status: Feedback Appreciated".

Just a nit: 

https://github.com/cncf/glossary/blob/ac0d758ff7c09394df34a7cc78d7d8eebbda73d0/content/en/_TEMPLATE.md?plain=1#L3